contracts/stake/src/state.rs

```diff
@@ -259,10 +260,8 @@ impl StakeState {
 
     /// Feeds the host with the stakes.
     pub fn stakes(&self) {
-        for (k, v) in self.stakes.iter() {
-            let pk = PublicKey::from_bytes(k).unwrap();
-            let stake_data = v.clone();
-            rusk_abi::feed((pk, stake_data));
+        for (_k, (stake_data, pk)) in self.stakes.iter() {
```

Comment:
Can you do `for (stake_data, pk) in self.stakes.values()`?

Just asking because I'm not sure if it's available in btree,